game/ui/main/pregame/pregame.lua

```diff
@@ -1040,18 +1040,52 @@ end
 local destroyPetWidgets
 local populatePetList
 
-local popularHeroList = {1, 2}
-local function IsHeroPopular(hero_Index)
-    for n, value in ipairs(popularHeroList) do
-        if (value == hero_Index) then
+local popularHeroList = 
+{
+    m_Array = {},
+    
+    DoesArrayContain = function (self, arraySearch, elementToFind)
```

Comment:
Perhaps more usable would be a .Any method, which takes an array, and a func which can return true/false for equality. Such a function likely exists.. somewhere..